packages/core/content-type-builder/admin/src/components/DataManagerProvider/utils/cleanData.js

```diff
@@ -94,6 +94,11 @@ const formatAttributes = (attributes, mainDataUID) => {
       acc[name] = removeNullKeys(formattedRelationAttribute);
     }
 
+    if (currentAttribute.customField) {
```

Comment:
Can you update the tests with your new statement?

Reply:
https://github.com/strapi/strapi/pull/14465/commits/a21e4a683a0e388c7175909294b07d3d1ffb2d8c